cmd/gopass-summon-provider/summon-provider_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	//"io"
+	"os"
+	//"strings"
+	"testing"
+
+	"github.com/fatih/color"
+	"github.com/gopasspw/gopass/internal/gptest"
+	"github.com/gopasspw/gopass/internal/out"
+	"github.com/gopasspw/gopass/internal/termio"
+	//"github.com/gopasspw/gopass/pkg/ctxutil"
+	"github.com/gopasspw/gopass/pkg/gopass/apimock"
+	//"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+
+func TestSummonProviderOutput(t *testing.T) {
+	ctx := context.Background()
+	act := &gc{
+		gp: apimock.New(),
+	}
+	require.NoError(t, act.gp.Set(ctx, "foo", &apimock.Secret{Buf: []byte("bar")}))
+
+	stdout := &bytes.Buffer{}
+	out.Stdout = stdout
+	color.NoColor = true
+	defer func() {
+		out.Stdout = os.Stdout
+		termio.Stdin = os.Stdin
+	}()
+
+	c := gptest.CliCtx(ctx, t)
```

Comment:
Something like this should work a be sufficient for this test:

```
assert.NoError(t, act.Get(gptest.CliCtx(ctx,t, "foo")))
assert.Equal(t, "bar", buf.String())
```

Reply:
refactored